powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehavior.java

```diff
@@ -784,54 +790,32 @@ private static String generateTotpSeed(final OperationCreateRequest request, fin
         return null;
     }
 
-    private boolean proximityCheckPassed(final OperationEntity operation, final OperationApproveRequest request) {
+    private static boolean proximityCheckPassed(final ProximityCheckResult proximityCheckResult) {
+        return proximityCheckResult == ProximityCheckResult.SUCCESS || proximityCheckResult == ProximityCheckResult.DISABLED;
```

Comment:
Can we log the disabled proximity check in a different way than when it is disabled?

Reply:
Good observation, we should probably just log the enum value there.

Reply:
The enum value is already in the audit record `Operation approved with ID: bc4e8032-4dc5-4741-b4a4-0c5cf188db9b`
```json
{
  "id": "bc4e8032-4dc5-4741-b4a4-0c5cf188db9b",
  "userId": "TestUserV31_f2d593e7-08e8-41e7-9310-96361a74050f",
  "appId": "PA_Tests",
  "status": "APPROVED",
  "additionalData": {
    "proximity_otp": "89569078"
  },
  "failureCount": 0,
  "proximityCheckResult": "SUCCESS",
  "currentTimestamp": "2023-05-23T14:40:26.893+00:00"
}
```
So, we may also expect `"proximityCheckResult": "DISABLED"`. Is that acceptable?